index.html

```diff
@@ -0,0 +1,767 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Building with Responsive Design</title>
+    <script src="https://kit.fontawesome.com/e252d07dd4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div class="top-nav grid">
+        <div class="news d-flex">
+          <div class="nav-item items1">
+            <a class="nav_a" href="#">News</a>
+          </div>
+          <div class="nav-item items2">
+            <a class="nav_a" href="#">Events</a>
+            <i class="fas fa-sort-down"></i>
+          </div>
+          <div class="nav-item items3">
+            <a class="nav_a" href="#">Business</a>
+            <i class="fas fa-sort-down"></i>
+          </div>
+          <div class="nav-item items4"><a class="nav_a" href="#"></a></div>
+          <div class="nav-item items5">
+            <a class="nav_a" href="#">Spaces</a>
+          </div>
+          <div class="nav-item items6">
+            <a class="nav_a" href="#">Terms & Conditions</a>
+          </div>
+        </div>
+        <div class="about d-flex">
+          <div class="nav-item items1"><a class="nav_a" href="#">About</a></div>
+          <div class="nav-item items2">
+            <a class="nav_a" href="#">Advertise</a>
+          </div>
+          <div class="nav-item items3"><a class="nav_a" href="#">Jobs</a></div>
+          <div class="nav-item items4">
+            <a class="nav_a" href="#">Contact</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="sticky_wrapper grid">
+      <div class="sticky_nav_bar grid">
+        <div class="latest1 grid aic">
+          <ul class="main">
+            <li class="first_one"><a class="gray" href="#"> LATEST</a></li>
+            <li><a class="gray" href="#"> HARD FORK</a></li>
+            <li><a class="gray" href="#"> PLUGED</a></li>
+            <li><a class="gray" href="#"> README</a></li>
+            <li><a class="gray" href="#"> GROWTH QUARTERS</a></li>
+            <li><a class="gray just_go" href="#"> SHIFT</a></li>
+            <li><a class="gray just_go" href="#"> NEURAL</a></li>
+            <li><img class="logo_img" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="TNW LOGO" /></li>
```

Comment:
**On Small And Medium Screens**

- [ ] The Image Should always display 

- [ ] On medium it should take a part of the left side

| Screen | Yours | Original |
| --- | --- | --- |
| **Small** | ![image](https://user-images.githubusercontent.com/11758151/93010468-d10d0100-f584-11ea-9b60-ac72f3dffbb8.png) | ![image](https://user-images.githubusercontent.com/11758151/93010462-c18db800-f584-11ea-847c-d4de8cf05be1.png) |
| **Medium** | ![image](https://user-images.githubusercontent.com/11758151/93010450-9f943580-f584-11ea-821e-1fb52e21a513.png) | ![image](https://user-images.githubusercontent.com/11758151/93010454-a9b63400-f584-11ea-95c3-097809429ca4.png) |